_posts/2021-11-18-introducing-opensearch-benchmark.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "Introducing OpenSearch Benchmark"
+authors:
+  - achitojha
+  - kyledvs
+date: 2021-11-17 01:01:01 -0700
+categories:
+  - update
+---
+
+I would like to introduce OpenSearch Benchmark - a community driven, open source fork of [Rally](https://esrally.readthedocs.io/en/stable/), a performance testing tool for Opensearch. The mission of the project is to provide a convenient and reliable performance testing framework for the OpenSearch community. OpenSearch Benchmark intends to be compatible with OpenSearch and Amazon OpenSearch Service. In alignment with other OpenSearch projects, “OpenSearch Benchmark” has been released and will be maintained under the Apache License, Version 2.0 (ALv2). 
+
+#### Why is this important for the OpenSearch community?
+As a user of OpenSearch you probably want to understand how various workloads perform across different settings, host types, and cluster configurations. OpenSearch Benchmark will enable you to use resources efficiently at optimal cost while having a good sense of what’s needed to serve their customers best. A performance testing mechanism also empowers you to run periodic benchmarks which can be used to identify performance regressions and improve performance. These reasons push the need for a performance testing mechanism for OpenSearch.
+
+#### Sounds interesting? Here’s what you can do!
+We encourage you to join us and be a part of the growing OpenSearch Benchmark community! Please install [OpenSearch Benchmark](https://github.com/opensearch-project/opensearch-benchmark/blob/main/README.md) and download the source [here](https://github.com/opensearch-project/opensearch-benchmark/). We’d like you to participate in the ongoing discussions and if you want to request any changes(bugs, feature requests), please create an issue [here](https://github.com/opensearch-project/opensearch-benchmark/issues/new/choose).
```

Comment:
...view the source on the OpenSearch Benchmark Github repository.

Rewrite last sentence to indicate that we want contributors as well.